### PR TITLE
review pr

### PR DESCRIPTION
**Pull request by dascalubogdan**

### CODE REVIEW

**Review by dascalubogdangmail — CHANGES_REQUESTED** [review]
change stuff

File: WebApplication1.Tests/UnitTest2.cs [hunk 1]
```diff
@@ -14,5 +14,12 @@ public void TestMethod2()
             HomeController c = new HomeController();
             Assert.IsNotNull(c.Index());
         }
+
+        [TestMethod]
```
Comment (dascalubogdangmail) [inline WebApplication1.Tests/UnitTest2.cs:18]: 2 comment

**Review by dascalubogdangmail — CHANGES_REQUESTED** [review]
222

File: WebApplication1.Tests/UnitTest2.cs (hunk 1 above)
Comment (dascalubogdan) [reply WebApplication1.Tests/UnitTest2.cs:18]: ok ok

**Review by dascalubogdangmail — APPROVED** [review]

**Review by dascalubogdangmail — APPROVED** [review]

**Pull request merged**